operators/operator_sdk/osdk-complying-with-psa.adoc

```diff
@@ -16,6 +16,9 @@ If your Operator project does not require escalated permissions to run, you can
 
 For more information, see xref:../../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Understanding and managing pod security admission].
 
+// About pod security admission
+include::modules/security-context-constraints-psa-about.adoc[leveloffset=+1]
```

Comment:
FYI @michaelryanpeter. I'm looking to add some more basic info about PSA. Since you included some of the other PSA modules in your operator docs, I was thinking you might want this as well. Let me know if not! 

(I did reuse one of the sentences from your assembly since it was perfect. I think it's far enough away that it shouldn't be too big of an issue, but again, let me know if you want me to make any modifications)

Reply:
@bergerhoffer I think this is a great addition to the Operator/PSA docs!